docs: Studio setup to include `&studio_unlock` (#2527)
zmkfirmware/zmk: docs/docs/features/studio.md

ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.

## Keymap Changes

> To unlock you keyboard to allow ZMK Studio to make changes, you'll need to add a [`&studio_unlock`](../keymaps/behaviors/studio-unlock.md) binding to the keymap in some unused location.

Reviewer: ```suggestion
To unlock your keyboard to allow ZMK Studio to make changes, you'll need to add a [`&studio_unlock`](../keymaps/behaviors/studio-unlock.md) binding to the keymap.
```
"Unused location" seems unneded here since user can decide whether to move things around, etc.
